RELEASE.md

```diff
@@ -37,6 +37,7 @@
 * Added the `--addons` flag to the `kedro new` command.
 * Added the `--name` flag to the `kedro new` command.
 * Removed `kedro run` flags `--node`, `--tag`, and `--load-version` in favour of `--nodes`, `--tags`, and `--load-versions`.
+* Removed the custom Kedro syntax for `--params`, use the OmegaConf syntax instead.
```

Comment:
I think we should mention this in the migration section, explaining that users should update their syntax from `:` to `=`.